src/test/groovy/transform/stc/BugsSTCTest.groovy

```diff
@@ -938,4 +938,22 @@ Printer
             }
         '''
     }
+
+    // GROOVY-9951
+    void testInnerImmutablePOJO() {
+        assertScript '''
+            import groovy.transform.Immutable
+            import groovy.transform.stc.POJO
+
+            class Outer {
+                @Immutable @POJO
```

Comment:
AFAIR, `@POJO` should be used with `@CompileStatic`.

Reply:
This test runs under STC and SC.

Reply:
OK